![Illustration of silhouetted heads](mentalhealth.jpg)

Does going to university in a different country affect your mental health? A Japanese international university surveyed its students in 2018 and published a study the following year that was approved by several ethical and regulatory boards.

The study found that international students have a higher risk of mental health difficulties than the general population. Explore the `students` data using PostgreSQL to find out if this is true and see if the length of stay is a contributing factor.

Here is a data description of the fields you may find helpful. The full dataset is in one table with 50 fields and, according to the survey, 268 records. Each row is a student.

| Field Name    | Description                                      | 
| ------------- | ------------------------------------------------ |
| inter_dom     | Types of students                                |
| japanese_cate | Japanese language proficiency                    | 
| english_cate  | English language proficiency                     |
| academic      | Current academic level                           | 
| age           | Current age of student                           |
| stay          | Current length of stay in years                  |
| todep         | Total score of depression (PHQ-9 test)           |
| tosc          | Total score of social connectedness (SCS test)   |
| toas          | Total score of Acculturative Stress (ASISS test) |

Your task will be to do the following exploratory analysis:

- Count the number of all records, and all records per student type
- Filter the data to see how it differs between the student types
- Find the summary statistics of the diagnostic tests for all students
- Summarize the data for international students
- See if length of stay impacts the test scores

### **1)Count the number of all records, and all records per student type**

In [4]:
SELECT inter_dom,
    COUNT(*) AS total_records
FROM students
GROUP BY inter_dom;

inter_dom  total_records
0       Dom             67
1                       18
2     Inter            201

- **There are 67 domestic students**
- **There are 201 international students**

### **2.1)Filter the data to see how it differs between the student types(International)**

In [9]:
SELECT ROUND(AVG(age),0) AS Average_Age,
    ROUND(AVG(stay),0) AS Average_Stay,
    MIN(japanese_cate) AS Japanese_Proficiency,
    MIN(english_cate) AS English_Proficiency,
    ROUND(AVG(todep),0) AS Average_Todep_Score,
    ROUND(AVG(tosc),0) AS Average_Tosc_Score,
    ROUND(AVG(toas),0) AS Average_Toas_Score
FROM students
WHERE inter_dom = 'Inter';
    

average_age  average_stay  ... average_tosc_score average_toas_score
0           21             2  ...                 37                 76

[1 rows x 7 columns]

### **2.2)Filter the data to see how it differs between the student types(Domestic)**

In [10]:
SELECT ROUND(AVG(age),0) AS Average_Age,
    ROUND(AVG(stay),0) AS Average_Stay,
    MIN(japanese_cate) AS Japanese_Proficiency,
    MIN(english_cate) AS English_Proficiency,
    ROUND(AVG(todep),0) AS Average_Todep_Score,
    ROUND(AVG(tosc),0) AS Average_Tosc_Score,
    ROUND(AVG(toas),0) AS Average_Toas_Score
FROM students
WHERE inter_dom = 'Dom';

average_age  average_stay  ... average_tosc_score average_toas_score
0           20             2  ...                 38                 63

[1 rows x 7 columns]

### **3)Find out the summary statistics of the diagnostic tests for all students**

In [12]:
SELECT MIN(todep) AS min_PHQ,
    MAX(todep) AS max_PHQ,
    ROUND(AVG(todep),2) AS avg_PHQ
FROM students;

min_phq  max_phq  avg_phq
0        0       25     8.19

In [13]:
SELECT MIN(tosc) AS min_SCS,
    MAX(tosc) AS max_SCS,
    ROUND(AVG(tosc),2) AS avg_SCS
FROM students;

min_scs  max_scs  avg_scs
0        8       48    37.47

In [14]:
SELECT MIN(toas) AS min_ASSIS,
    MAX(toas) AS max_ASSIS,
    ROUND(AVG(toas),2) AS avg_ASSIS
FROM students;

min_assis  max_assis  avg_assis
0         36        145      72.38

### **4)Summarize the data for international students only**

In [15]:
SELECT MIN(toas) AS min_ASSIS,
    MAX(toas) AS max_ASSIS,
    ROUND(AVG(toas),2) AS avg_ASSIS,
    MIN(tosc) AS min_SCS,
    MAX(tosc) AS max_SCS,
    ROUND(AVG(tosc),2) AS avg_SCS,
    MIN(todep) AS min_PHQ,
    MAX(todep) AS max_PHQ,
    ROUND(AVG(todep),2) AS avg_PHQ
FROM students
WHERE inter_dom = 'Inter'
GROUP BY inter_dom;

min_assis  max_assis  avg_assis  min_scs  ...  avg_scs  min_phq  max_phq  avg_phq
0         36        145      75.56       11  ...    37.42        0       25     8.04

[1 rows x 9 columns]

### **5)See the impact of length of stay**

In [16]:
SELECT stay,
     ROUND(AVG(todep),2) AS average_phq,
     ROUND(AVG(tosc),2) AS average_scs,
     ROUND(AVG(toas),2) AS average_as
FROM students
WHERE inter_dom = 'Inter'
GROUP BY stay
ORDER BY stay DESC;
    

stay  average_phq  average_scs  average_as
0    10        13.00        32.00       50.00
1     8        10.00        44.00       65.00
2     7         4.00        48.00       45.00
3     6         6.00        38.00       58.67
4     5         0.00        34.00       91.00
5     4         8.57        33.93       87.71
6     3         9.09        37.13       78.00
7     2         8.28        37.08       77.67
8     1         7.48        38.11       72.80